### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by SnabbSpelare** [elifoster/ChaoticKarma#8]
I'm sorry but does this do anything on it's own?

**Comment by elifoster** [comment]
Yes it has some basic features but was intended as an API to be used by other devs, perhaps in themed packs etc. You can see what karma triggers, karma based perks, and random events it has here, as well as a modding guide for it https://ftb.fandom.com/wiki/ChaoticKarma
